docs/make.jl

```diff
@@ -8,12 +8,24 @@ const generated_dir = joinpath(@__DIR__, "src", "generated")
 using Catlab
 
 @info "Building Literate.jl docs"
+
+# Define config if not being compiled on recognized service
+config = Dict{String,String}()
+env_set = haskey(ENV, "HAS_JOSH_K_SEAL_OF_APPROVAL") ||
+   haskey(ENV, "GITHUB_ACTIONS") ||
+   haskey(ENV, "GITLAB_CI")
+
+if !env_set
+    config["nbviewer_root_url"] = "https://nbviewer.jupyter.org/github/epatters/Catlab.jl/blob/gh-pages/v0.5.2"
```

Comment:
Can this link point to a tag where the version is not hard-coded, something like `gh-pages` or `gh-pages/master`? Not sure whether these work.

Reply:
Replaced the static version number with dev